src/profile/components/errorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory, useLocation } from 'react-router-dom';
+import { Button } from 'hds-react';
+
+import Notification from '../../../common/copyOfHDSNotification/Notification';
+import PageLayout from '../../../common/pageLayout/PageLayout';
+import queryParamsToObject from '../../../common/helpers/queryParamsToObject';
+import styles from './ErrorPage.module.css';
+import authService from '../../../auth/authService';
+
+export type ErrorPageQueryParams = {
+  message?: string;
+  title?: string;
+  hideLoginButton?: string;
+  hideFrontPageButton?: string;
+};
+
+export const errorPagePath = '/error';
+
+function ErrorPage(): React.ReactElement {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const location = useLocation();
+  const {
+    message,
+    title,
+    hideLoginButton,
+    hideFrontPageButton,
+  } = queryParamsToObject<ErrorPageQueryParams>(location.search);
+  const notificationMessage = message || t('notification.defaultErrorText');
+  const notificationTitle = title || t('notification.defaultErrorTitle');
+  const isAuthenticated = authService.isAuthenticated();
+
+  return (
+    <PageLayout title={title} data-testid="error-page-layout">
```

Comment:
Should the `PageLayout`'s title also be set to `notificationTitle`, in order to use the default title when needed?

Reply:
Thx! Changed this.